src/OrchardCore.Modules/OrchardCore.Title/Views/TitlePart.Edit.cshtml

```diff
@@ -1,16 +1,21 @@
 @model TitlePartViewModel
+@using OrchardCore.ContentManagement.Metadata
+@using OrchardCore.ContentManagement.Metadata.Settings
 @using OrchardCore.ContentLocalization
 @using OrchardCore.Localization
 @using OrchardCore.Title.Models
 @using OrchardCore.Title.ViewModels;
 
+@inject IContentDefinitionManager ContentDefinitionManager
+
 @{
     var culture = await Orchard.GetContentCultureAsync(Model.TitlePart.ContentItem);
+    var displayName = ContentDefinitionManager.GetPartDefinition("TitlePart").DisplayName();
 }
 @if (Model.Settings?.Options != TitlePartOptions.GeneratedHidden)
 {
     <div class="form-group" asp-validation-class-for="Title">
-        <label asp-for="Title">@T["Title"]</label>
+        <label asp-for="Title">@displayName</label>
         <input asp-for="Title" class="form-control content-preview-text content-caption-text" disabled="@(Model.Settings?.Options == TitlePartOptions.GeneratedDisabled)" autofocus="autofocus" dir="@culture.GetLanguageDirection()" />
         <span asp-validation-for="Title"></span>
         @if (String.IsNullOrWhiteSpace(Model.Settings?.Pattern) && Model.Settings?.Options == TitlePartOptions.Editable)
```

Comment:
Maybe another issue, but the hint below should, in my opinion, be customizable. We would often avoid using the TitlePart in our content types because we could not name it / modify the hint.

Reply:
I think someone else already started another PR to address the hint edit